soci/soci_index.go

```diff
@@ -68,6 +68,7 @@ const (
 	// [soci-snapshotter] NOTE: this is a duplicate of fs/layer/layer.go so that the SOCI package and the snapshotter can
 	// be independent (and SOCI could be split out into it's own module in the future).
 	whiteoutOpaqueDir = ".wh..wh..opq"
+	DisableXAttrsTrue = "true"
```

Comment:
Small nit. Let's reduce the scope of this until it needs to be exported (if ever).